doc/nrf/config_and_build/bootloaders_and_dfu/fw_update.rst

```diff
@@ -329,18 +351,63 @@ If this image has, in order of precedence, a *major*, *minor*, or *revision* val
 .. note::
 
    The optional label or build number specified after the ``+`` character is ignored when evaluating the version.
-   An existing application image with version ``0.1.2+3`` can be overwritten by an uploaded image with ``0.1.2+2``, but not by one with ``0.1.1+2``.
+   For example, an existing application image with version ``0.1.2+3`` can be overwritten by an uploaded image with ``0.1.2+2``, but not by one with ``0.1.1+2``.
 
-Hardware-based downgrade prevention
------------------------------------
+.. _ug_fw_update_downgrade_protection_hw:
+.. _bootloader_monotonic_counter:
 
-Hardware-based downgrade prevention is implemented with a counter value stored in the *provision* partition of the non-volatile storage.
+Hardware-based downgrade protection
+===================================
 
-See the :ref:`bootloader_monotonic_counter` section for how to enable and configure this feature.
+.. bootloader_monotonic_counter_start
+
+You can implement hardware-based downgrade protection using a non-volatile monotonic counter.
+
+Counter updates are written to slots in the *Provision* area, with each new counter update occupying a new slot.
+For this reason, the number of counter updates, and therefore firmware version updates, is limited.
+The *Provision* is a partition in non-volatile memory, and its location can be found using :ref:`Partition Manager <partition_manager_report>`.
+
+Using a counter is optional and can be configured for the application using configuration options.
+You can also configure the supported number of updates, but the number is limited by the size of the *Provision* area and how much of that area is taken up by other features, like public key hashes.
+In addition, you can configure what firmware version of the image you want to boot.
+
+.. bootloader_monotonic_counter_end
+
+Special handling is needed when updating the S1 variant of an image.
+See :ref:`ug_bootloader_adding_presigned_variants` for details.
+
+.. _ug_fw_update_hw_downgrade_nsib:
+
+Using |NSIB|
+============
+
+.. bootloader_monotonic_counter_nsib_start
+
+For |NSIB|, the configuration options for implementing the anti-rollback protection with monotonic counter in your application are the following:
+
+* :kconfig:option:`CONFIG_SB_MONOTONIC_COUNTER`
+* :kconfig:option:`CONFIG_SB_NUM_VER_COUNTER_SLOTS`
+
+.. bootloader_monotonic_counter_nsib_end
+
+To set options for child images, see the :ref:`ug_multi_image_variables` section.
+
+.. _ug_fw_update_hw_downgrade_mcuboot:
+
+Using MCUboot
+=============
+
+For MCUboot, the configuration options for implementing the anti-rollback protection with monotonic counter in your application are the following:
```

Comment:
Tricky thing is that the feature is for MCUboot child image, but the configuration is to be set for the application parent image. Can I suggest that we rephrase this:
```suggestion
To enable anti-rollback protection with monotonic counter for MCUboot, set the following configurations in the application (parent image):
```